data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py

```diff
@@ -74,7 +55,8 @@ def cstr(s, color='black'):
 
 
 '''.format(DATASET_ID=dataset_id)
-number_of_rows_affected = pd.read_gbq(number_of_rows_affected,dialect="standard")
+number_of_rows_affected = pd.read_gbq(number_of_rows_affected,
+                                      dialect="standard")
```

Comment:
Are these formatting changes due to yapf?

Reply:
Yes. Most of the changes were coming yapf.